syntax = "proto3";

package confidence.wasm;

option java_package = "com.spotify.confidence.wasm";
option java_multiple_files = false;
option java_outer_classname = "Messages";
option go_package = "github.com/spotify/confidence-resolver/openfeature-provider/go/confidence/internal/proto/wasm";

message Void {}

message SetResolverStateRequest {
    bytes state = 1;
    string account_id = 2;
}

message Request {
    bytes data = 1;
}

message Response {
    oneof result {
        bytes data = 1;
        string error = 2;
    }
}
